### DISCUSSION TITLE
[Feature Request] Set Rules Info Box

### DISCUSSION THREAD

**Discussion by BahamutVoid** [RetroAchievements/RAWeb#2559] [Ideas]
Games are complex. Some have cheat codes, known exploits and other oddities which makes balancing achievements difficult. Oftentimes we disallow them through logic and state this in an achievement's description but when it applies to a whole set this can lead to either repeating the restriction many times or expecting the user to know it isn't allowed.

We currently have two rules that cover this situation, rules [#2](https://docs.retroachievements.org/guidelines/users/global-leaderboard-and-achievement-hunting-rules.html#_2-do-not-earn-achievements-in-a-way-that-contradicts-what-the-description-asks) and [#10](https://docs.retroachievements.org/guidelines/users/global-leaderboard-and-achievement-hunting-rules.html#_10-do-not-use-in-game-cheat-codes-unless-it-is-specified-as-allowed-by-the-developer). Cheat codes are by default disallowed when earning achievements except when stated otherwise. If an achievement is unlocked when one has been used, it is a bug and should be reported with the ticket system. If not stated in the achievement's description it can be added to the forum topic, achievement or game comments and it's assumed the user will see them.

This assumption has led to confusion in the past. We should improve our transparency by creating a Set Rules box on a game's page.
This will also complement any future tagging system such as [#1694](https://github.com/RetroAchievements/RAWeb/discussions/1694) by reducing redundancies even further to prevent as much clutter as possible.

Example information to include could be:
- What is allowed
- What is disallowed
- Set specific rules (particularly subsets)
- Game specific notes
- Specific settings or bios requirements
- Emulator/core the set was developed/tested with

Mockup:
![image](https://github.com/user-attachments/assets/8861f0a5-2037-45a5-b22b-2b1d78eee169)

In the future I would like to see this added to emulators as well to benefit those who don't visit the site. Just like on a game's page, it would be above the achievement list.

Mockup:
![image](https://github.com/user-attachments/assets/cf80e685-a56c-41fd-9e3c-ae6c55f7b5b7)

**Comment by Jamiras** [comment]
Related: #708. Though I think this is more fleshed out.
